packages/docs/components/molecules/AnchorNav/stories/app.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable max-classes-per-file */
+import { Base, createApp } from '@studiometa/js-toolkit';
+import {
+  AnchorNav as AnchorNavCore,
+  AnchorNavLink,
+  AnchorNavTarget,
+} from '@studiometa/ui';
+
+/**
+ *
+ */
+class AnchorNav extends AnchorNavCore {
+  static config = {
+    components: {
+      AnchorNavLink,
+      AnchorNavTarget,
```

Comment:
**suggestion:** Use the `AnchorNav` class from the package directly

Both `AnchorNavLink` and `AnchorNavTarget` components are already defined as children components of `AnchorNav`, we do not have to redefine them here.